server/src/test/java/org/candlepin/policy/js/pool/PoolHelperTest.java

```diff
@@ -14,9 +14,15 @@
  */
 package org.candlepin.policy.js.pool;
 
-import static org.junit.Assert.*;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNotEquals;
+import static org.junit.Assert.assertTrue;
+import static org.mockito.AdditionalAnswers.returnsFirstArg;
+import static org.mockito.Matchers.anyListOf;
 import static org.mockito.Matchers.eq;
-import static org.mockito.Mockito.*;
```

Comment:
I would just keep these as wildcard imports.  It's kind of a pain when you are writing additional tests and use a new assert to have to jump to the imports and import it.  If Eclipse is auto-removing your wildcards for static imports, you can go to Window -> Preferences -> Java -> Code Style -> Organize Imports and set "Number of static imports needed for .*" to 1 or 2.  (I have mine at two because I figure if I need more than one static import, I'm probably going to want access to all of them).